DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fpga_template/README.md

```diff
@@ -0,0 +1,92 @@
+# FPGA Template
+
+This project serves as a template for Intel® oneAPI FPGA designs. 
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
+| Hardware                          | Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX) <br> Intel® FPGA 3rd party / custom platforms with oneAPI support <br> *__Note__: Intel® FPGA PAC hardware is only compatible with Ubuntu 18.04*
+| Software                          | Intel® oneAPI DPC++ Compiler <br> Intel® Quartus Prime Pro Edition <br> Siemens® Questa® Intel® FPGA Starter Edition
+| What you will learn               | Best practices for creating and managing a oneAPI FPGA project
+| Time to complete                  | 10 minutes
+
+## Purpose
+
+Use this project as a starting point when you build designs for the Intel® oneAPI FPGA compiler. It includes a CMake build system to abstract away the need to know all the various command-line flags for the `icpx` compiler, and a simple single-source design to serve as an example. You can customize the build flags by modifying the top part of `src/CMakeLists.txt`. If you want to pass additional flags to Intel DPC++ compiler, you can change the `USER_FLAGS` and `USER_HARDWARE_FLAGS` variables defined in `src/CMakeLists.txt`. 
+
+| Variable              | Description                                                                                                                                                                       |
+|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `USER_HARDWARE_FLAGS` | These flags apply only to flows that generate FPGA hardware (i.e. report, simulation, hardware). You can specify flags such as `-Xsclock` or `-Xshyper-optimized-handshaking=off` |
+| `USER_FLAGS`          | These flags apply to all flows, including emulation. You can specify flags such as `-v` or define macros such as `-DYOUR_OWN_MACRO=3`                                             |
+
+```bash
+###############################################################################
+### Customize these build variables
+###############################################################################
+set(SOURCE_FILE vector_add.cpp)
+set(TARGET_NAME vector_add)
+
+# When targeting a board, the `FPGA_BOARD` will be targeted. Options are:
+#   intel_s10sx_pac:pac_s10
+#   intel_s10sx_pac:pac_s10_usm
+#   intel_a10gx_pac:pac_a10
+set(FPGA_BOARD "intel_s10sx_pac:pac_s10_usm")
```

Comment:
OK i think I like this. My intention had been to set a 'board target' and a 'device target' and then let the user choose to build for a board or build for an IP, but i suppose we can restrict the build targets and select 'board' or 'device' based off whatever the target the user chooses is. makefile targets would be:
- `fpga_emu`
- `report`
- `fpga_sim`
- `fpga_ip_export` (add once we support IP authoring; this will effectively just be the same as `report`, but with `-fsycl-device-code-split=per_kernel`).
- `fpga` (will run quartus for both IP Authoring and full-system)